experiments/compositions/composition/tests/testcases/main_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"google.com/composition/tests/kind"
+	"google.com/composition/tests/utils"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	images *string = flag.String("images", "gcr.io/allotrope-barni/composition:v0.0.1.alpha,gcr.io/allotrope-barni/manifests-inline:v0.0.1.alpha,gcr.io/allotrope-barni/expander-jinja2:v0.0.1.alpha", "images")
+)
+
+func RegisterImages(kc kind.KindCluster) error {
+	for _, image := range strings.Split(*images, ",") {
+		err := kc.LoadImage(image)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// TestMain - umbrella test that runs all test cases
+func TestMain(m *testing.M) {
+	flag.Parse()
+
+	rand.Seed(time.Now().UnixNano())
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Please use klogr

Reply:
Do you mind showing the change ? 
is it klog.SetLogger() ? 
or logf.SetLogger(klog.New()) ?